ibis/odbc/impala.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2015 Cloudera Inc
```

Comment:
Actually, before you do that ... @wesm How should we handle new files? Is there any copyright needed?